.semgrep.yml

```
@@ -0,0 +1,8 @@
+rules:
+  - "p/default"      # Base security rules
+  - "p/react"        # React-specific rules
+  - "p/nextjs"       # Next.js framework rules
+  - "p/typescript"   # TypeScript language rules
+  - "p/javascript"   # JavaScript language rules
+  - "p/security-audit"  # Security best practices
+  - "p/owasp-top-ten"   # OWASP top 10 vulnerabilities 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)